src/get_opening_hours.spec.ts

```diff
@@ -74,7 +76,16 @@ describe('getOpeningHours()', () => {
     ${invalidString2}
     ${invalidString3}
     ${invalidString4}
+    ${invalidString5}
   `('throws an error given $input', ({ input }) => {
     expect(() => getOpeningHours(input)).toThrow();
   });
+
+  describe('absolute days', () => {
```

Comment:
Move this (and all other equally named) group up, above tests regarding invalid strings. Also since there's just one test, maybe not wrap it with describe().